README.md

```diff
@@ -43,7 +46,36 @@ And then access the server in https://localhost:8080/
 
 # How WasmerJS works
 
-Wasmer Winter is powered by [SpiderMonkey](https://spidermonkey.dev/) and [Axum](https://github.com/tokio-rs/axum)
+WinterJS is powered by [SpiderMonkey](https://spidermonkey.dev/), [Spiderfire](https://github.com/Redfire75369/spiderfire) and [hyper](https://hyper.rs/)
 to bring a new level of awesomeness to your Javascript apps.
 
-To compile the app to WebAssembly WinterJS is using the [WASIX](https://wasix.org) standard.
+To compile the app to WebAssembly WinterJS is using the [WASIX](https://wasix.org) standard. Please note that compiling to WASIX is currently an involved process. We recommend using precompiled versions from [`wasmer/winterjs`](https://wasmer.io/wasmer/winterjs).
+
+# WinterCG API Compatibility
+
+This section will be updated as APIs are added/fixed.
+If an API is missing from this section, that means that it is still not implemented.
+
+The following words are used to describe the status of an API:
+
+* Pending - The API is not implemented yet.
+* Partial - The API is implemented but not fully compliant with the spec and/or there are known limitations.
+* Stable - The API is implemented and fully compliant with the spec. This does not account for potential undiscovered implementation errors in the native code.
+
+|API|Status|Notes|
+|:-:|:-:|:--|
+|`console`|Stable|
+|`fetch`|Partial|`Request`, `Response` and `ReadableStream` need to be stabilized before `fetch` can be considered Stable
+|`URL`|Stable|
+|`URLSearchParams`|Stable|
+|`Request`|Partial|`ReadableStream` needs to be stabilized before `Request` can be considered Stable<br/>Requests cannot be sent with `ReadableStream` bodies
+|`Headers`|Stable|
+|`Response`|Partial|`ReadableStream` needs to be stabilized before `Response` can be considered Stable.<br/>`Response.body` returns a Promise that resolves to a `ReadableStream` instead of returning a `ReadableStream` directly.
+|`Blob`|Partial|`Blob.stream()` is not implemented yet
+|`FormData`|Stable|
+|`TextDecoder`|Partial|Only UTF-8 is supported|
+|`TextEncoder`|Partial|Only UTF-8 is supported|
+|`ReadableStream`|Partial|Creating `ReadableStream`s with `type: 'bytes'` is not supported yet.<br/>Returning `ReadableStream`s to native code is not supported yet.
+|`atob`|Stable|
+|`btoa`|Stable|
+|`performance.now()`|Stable|
```

Comment:
This table is hard to read. Please use emojis near the Partial / Stable to represent and read it better